### PR TITLE
move es2016+ and stage 4 things to new tab

### PR DESCRIPTION
**Pull request by chicoxyzzy**
Previous discussion https://github.com/kangax/compat-table/issues/822

### CODE REVIEW

**Comment by chicoxyzzy** [comment]
I've choose name "Accepted" for new tab but I'm open to any other suggestions

File: data-esnext-accepted.js [hunk 1]
```diff
@@ -0,0 +1,1243 @@
+// exports browsers and tests
+// new browsers should only be added once they have at least a single 'true' result.
+exports.name = 'ES Next Accepted';
```
Comment (ljharb) [inline data-esnext-accepted.js]: I think a more accurate name here would be `ES 2016+` or `Finished ECMAScript Proposals`, per https://tc39.github.io/process-document/ - "Accepted" isn't a term that's usually used.

Comment (chicoxyzzy) [inline data-esnext-accepted.js]: @ljharb I think `ES2016+` is fine

**Comment by webbedspace** [comment]
Extra favour: move the `exports.browsers` object into a separate file `require`d by both esnext.js files. Will save us all in the long run.

**Comment by chicoxyzzy** [comment]
Should I do something else here?

**Pull request merged**